Modules/ITS/include/ITS/ITSChipStatusTask.h

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   ITSChipStatusTask.h
+/// \author My Name
+///
+
+#ifndef QC_MODULE_ITS_ITSITSCHIPSTATUSTASK_H
+#define QC_MODULE_ITS_ITSITSCHIPSTATUSTASK_H
+
+#include "QualityControl/TaskInterface.h"
+#include "Common/Utils.h"
+
+#include <TH1D.h>
+#include <TH2D.h>
+
+using namespace o2::quality_control::core;
+using namespace o2::quality_control_modules::common;
+
+namespace o2::quality_control_modules::its
+{
+
+class Stack
+{
+ private:
+  int size_max;
+  int current_element_id;
+
+ public:
+  std::vector<std::vector<int>> stack;
+  Stack() : size_max(0), current_element_id(0) {}
+  Stack(int nsizey, int nsizex) : size_max(nsizey), current_element_id(0)
```

Comment:
Not fundamental but probably you can invert `nsizey` with `nsizex` since when you define a pointer to this `class` you use `new Stack(nQCCycleToMonitor, ChipsBoundaryBarrels[i + 1] - ChipsBoundaryBarrels[i]);` but then the first element refers to the `x` axis of the TH2D while the second element refers to the `y` axis.